src/linear_algebra/general_linear_group.lean

```diff
@@ -143,10 +145,17 @@ variables {n : Type u} {R : Type v} [decidable_eq n] [fintype n] [linear_ordered
 /-- Formal operation of negation on general linear group on even cardinality `n` given by negating
 each element. -/
 instance : has_neg (GL_pos n R) :=
-⟨λ g, ⟨-g, begin
-    rw [mem_GL_pos, general_linear_group.coe_det_apply, units.coe_neg, det_neg,
-      nat.neg_one_pow_of_even (fact.out (even (fintype.card n))), one_mul],
-    exact g.prop,
+⟨λ g,
+   ⟨- g,
+  begin
+    simp only [mem_GL_pos, general_linear_group.coe_det_apply, units.coe_neg],
+    have := det_smul g (-1),
+    simp only [general_linear_group.coe_fn_eq_coe, one_smul, coe_fn_coe_base', neg_smul] at this,
+    rw this,
+    simp [nat.neg_one_pow_of_even (fact.out (even (fintype.card n)))],
+    have gdet := g.property,
+    simp only [mem_GL_pos, general_linear_group.coe_det_apply, subtype.val_eq_coe] at gdet,
+    exact gdet,
   end⟩⟩
```

Comment:
Did you mean to make this longer? It seems like you might have pasted some old changes into a new branch on top of master (in doing so reverting #12396 and likely some other changes), rather than using `git` to merge them cleanly.

Reply:
Oops I didn't mean that! I was trying to do a git merge but it was complaining so I just copy-pasted! let me try again